cli/flags.go

```diff
@@ -61,6 +61,21 @@ var (
 		Value: "10s",
 		Usage: "timeout period",
 	}
+	flRefreshIntervalMin = cli.StringFlag{
+		Name:  "engine-refresh-min-interval",
+		Value: "30s",
+		Usage: "set engine refresh minimum interval",
+	}
+	flRefreshIntervalMax = cli.StringFlag{
+		Name:  "engine-refresh-max-interval",
+		Value: "60s",
+		Usage: "set engine refresh maximum interval",
+	}
+	flRefreshRetry = cli.StringFlag{
```

Comment:
Thanks @jimmyxian. I use `StringFlag` to be consistent with other duration flags like flTTL.